src/domain/Tables/Table/subcomponents/ActionIconButtonGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { FontAwesomeIconButton, IFontAwesomeIconButtonProps } from '../../../../domain/Buttons/FontAwesomeIconButton/FontAwesomeIconButton'
+import { StyledFontAwesomeIconButtonWrapper } from '../services/style'
+
+const getActionsIconButtonGroup = (actions?: IFontAwesomeIconButtonProps[], name?: string) => {
```

Comment:
Technically not a component, but this is consistent with how it's working at the moment